src/Controls/src/Build.Tasks/CreateObjectVisitor.cs

```diff
@@ -575,53 +571,5 @@ IEnumerable<Instruction> PushValueFromLanguagePrimitive(TypeDefinition typedef,
 					break;
 			}
 		}
-
-		internal static IEnumerable<Instruction> RegisterSourceInfo(ILContext context, INode valueNode)
-		{
-			if (!context.DefineDebug)
```

Comment:
Do we still need the `DefineDebug` property? In my quick inspection, this appears to be the only place we read it.